scraping/scraping/spiders/zalando.py

```diff
@@ -62,9 +62,8 @@ def parse_SERP(self, response: SplashJsonResponse) -> Iterator[SplashRequest]:
             )
 
         # Pagination: Parse next SERP 'recursively'
-        next_page = response.css('[title="nächste Seite"]::attr(href)').extract_first()
-
-        if next_page:
```

Comment:
I would prefer to keep this structure (also the `else`) and only change how you find `next_page`.